### ISSUE TITLE
Push notification settings are not visible in Firefox.

### ISSUE THREAD

**Issue by pranavrajs** [chatwoot/chatwoot#1606]
I cannot see the push notification settings in Firefox. It looks like an issue with CSS.

**Comment by hartois** [comment]
You are about this?

Google Chrome:
![image](https://user-images.githubusercontent.com/11641656/104168983-ea796900-540f-11eb-9916-94a4eab639bd.png)

Should I see these settings in self-hosted installation?

**Comment by hartois** [comment]
Release: v1.11.1

**Comment by pranavrajs** [comment]
@hartois Have you configured the env variables? https://www.chatwoot.com/docs/environment-variables#push-notification

**Comment by hartois** [comment]
@pranavrajs Oh thank you! I missed it

**Comment by jcomo** [comment]
This might not be the same problem as @hartois was having, but I'm not seeing it in Firefox either. I'm using the cloud-hosted version (for good measure, I verified the vapid key is in `window.chatwootConfig`) and Firefox version 84.0.2

I _think_ the issue is in the Safari detection here: https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/routes/dashboard/settings/profile/NotificationSettings.vue#L255-L258. I admit I'm not well enough versed in CSS-fu to know how to interpret that selector, but my best guess is that it's matching for newer versions of Firefox. This should be a more robust browser check if that's the intent.

I was able to work around it by opening up developer tools, finding the div and unchecking the "display: none" rule. Then I could interact with it and enable push notifications.

![Screen Shot 2021-01-13 at 4 35 23 PM](https://user-images.githubusercontent.com/1068249/104528974-78df1d80-55bd-11eb-8abe-b9be702276c5.png)
